src/store/index.js

```diff
@@ -11,7 +11,9 @@ export default createStore({
 			username: '',
 			authToken: '',
 			profile: [],
-			projects: []
+			projects: [],
+			largeImages: [],
+			firstLargeImage: ''
```

Comment:
would it make sense for the firstLargeImage just be a getter on largeImages? e.g.

```
  getters: {
    firstLargeImage (state) {
      return state.largeImages[0].url
    }
  }
```

https://vuex.vuejs.org/guide/getters.html#getters

Reply:
done!